### PR TITLE
added to do bullet to horizon landmark

### PR DESCRIPTION
**Pull request by OwenHarveyBG**
I've noticed x y z...

### CODE REVIEW

File: src/content/docs/components/ns-landmark.mdx [hunk 1]
```diff
@@ -174,11 +174,19 @@ The purpose of the valley landmark is to present an overview of a product or ser
...
+- Use this on tier 1 pages - such as home page and overview pages
 :::
 
 :::dont
-- TODONTs.
+- Use long, wordy content. Instead, keep the user interested with a snappy overview.
+- Use images with a background.
+- Add a CTA if the page’s purpose is for users to consume the content.
+- Use the red decoration for something that is not a sale or promotion.
+- Use the slate decoration for something that is not a Black Friday messaging or promotion.
```
Comment (PtitBen) [inline src/content/docs/components/ns-landmark.mdx:189]:
```suggestion
- Use the slate decoration for something that is not a Black Friday messaging.
```

File: src/content/docs/components/ns-landmark.mdx [hunk 2]
```diff
@@ -174,11 +174,19 @@ The purpose of the valley landmark is to present an overview of a product or ser
...
+- Use a complementary image to add context to your product or service.
+- Use this on tier 1 pages - such as home page and overview pages
 :::
 
 :::dont
-- TODONTs.
+- Use long, wordy content. Instead, keep the user interested with a snappy overview.
+- Use images with a background.
+- Add a CTA if the page’s purpose is for users to consume the content.
+- Use the red decoration for something that is not a sale or promotion.
```
Comment (PtitBen) [inline src/content/docs/components/ns-landmark.mdx:188]:
```suggestion
- Use the red decoration for something that is not a sale.
```

File: src/content/docs/components/ns-landmark.mdx [hunk 3]
```diff
@@ -174,11 +174,19 @@ The purpose of the valley landmark is to present an overview of a product or ser
 | <span class="marker">E</span> | Decoration | Apply a colour background. The `decoration` values are restricted to `cyan`, `red`, `slate`. All other colours will default to `cyan` |
 
 :::do
-- TODOs.
+- Use at the top of the page
```
Comment (PtitBen) [inline src/content/docs/components/ns-landmark.mdx:177]:
```suggestion
- Use at the top of the page.
```

File: src/content/docs/components/ns-landmark.mdx [hunk 4]
```diff
@@ -174,11 +174,19 @@ The purpose of the valley landmark is to present an overview of a product or ser
...
+- Use for 'above-the-line' marketing messaging and promotions.
+- Keep the content concise.
+- Use a complementary image to add context to your product or service.
+- Use this on tier 1 pages - such as home page and overview pages
```
Comment (PtitBen) [inline src/content/docs/components/ns-landmark.mdx:181]:
```suggestion
- Use this on tier 1 pages - such as home page and overview pages.
```

**Review by PtitBen — COMMENTED** [review]
Removed the "or promotion" from the red and slate backgrounds to be strict like the brand guidelines are around the use of those colours.

(and a couple of full stops :p)

**Review by RobTobias123 — APPROVED** [review]
Approved - with one minor grammatical suggestion made

**Pull request merged**